Index.html

```diff
@@ -144,8 +144,8 @@ <h2>Survey</h2>
                     <label><input type="radio" value="no" name="radio">No</label>
                     <label><input type="radio" value="?" name="radio" checked>None of your business</label>
 
-                    <label>What is the meaning of life?</label>
-                    <textarea rows="10">Enter the correct answer here</textarea>
+                    <label for="life_meaning">What is the meaning of life?</label>
+                    <textarea rows="10" name="life_meaning" id="life_meaning" placeholder="Enter the correct answer here"></textarea>
```

Comment:
Missing label 'for' attribute and id attribute on corresponding control
Missing field name
Missing placeholder